internal/charmstore/server.go

```diff
@@ -103,6 +103,30 @@ type ServerParams struct {
 	// RunBlobStoreGC holds whether the server will run
 	// the blobstore garbage collector worker.
 	RunBlobStoreGC bool
+
+	// BlobStore holds the type of blobstore to use.
+	BlobStore string `yaml:"blobstore"`
+
+	// SwiftAuthURL holds the swift auth url, equivalent to OS_AUTH_URL.
+	SwiftAuthURL string `yaml:"swift-auth-url"`
```

Comment:
Can we put this off until later? I agree it will be good, but with Azure Blob Storage coming as a next blob store, it will let us consider how all that config will look instead of this config alone.